README.md

```diff
@@ -1,16 +1,16 @@
-# tinycolor
+# tinycolor2
 > TinyColor is a small library for Flutter color manipulation and conversion
 
 A port of [tinycolor2](https://github.com/bgrins/TinyColor) by [Brian Grinstead](https://github.com/bgrins)
 
 ## Getting Started
 
-A tinycolor receives a `Color` as parameter.
+A `TinyColor` receives a `Color` as parameter.
 
 ```dart
-import 'package:tinycolor/tinycolor.dart';
+import 'package:tinycolor2/tinycolor2.dart';
 
-final TinyColor = TinyColor(Colors.green);
+final TinyColor tinyColor = TinyColor(Colors.green);
 ```
 Now you can also use the package to extend the native Color class with all the same features, but simpler. To use extension update, make sure to change envieronment sdk version in pubspec like this: ` sdk: ">=2.6.0 <3.0.0"`
```

Comment:
`Color`

Reply:
Not that line. The Color in the text is not quoted.

Reply:
oh sorry